### PR TITLE
Use Goerli instead of Rinkeby for Cowswap tests

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

**Comment by moisses89** [comment]
Maybe `Use Goerli instead of Rinkeby for Cowswap tests` :sweat_smile:

**Comment by Uxio0** [comment]
> Maybe `Use Goerli instead of Rinkeby for Cowswap tests` 😅

Nice catch, done

File: gnosis/protocol/tests/test_gnosis_protocol_api.py [hunk 1]
```diff
@@ -14,7 +14,8 @@ class TestGnosisProtocolAPI(TestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.gnosis_protocol_api = GnosisProtocolAPI(EthereumNetwork.RINKEBY)
+        cls.gnosis_protocol_api = GnosisProtocolAPI(EthereumNetwork.GOERLI)
```
Comment (moisses89) [inline gnosis/protocol/tests/test_gnosis_protocol_api.py:17]: Could be goerli_gnosis_protocol_api? It's a little bit confused have mainnet and not goerli.

File: gnosis/protocol/tests/test_gnosis_protocol_api.py [hunk 2]
```diff
@@ -76,23 +77,21 @@ def test_get_fee(self):
         self.assertGreaterEqual(self.gnosis_protocol_api.get_fee(order), 0)
 
     def test_get_trades(self):
+        mainnet_order_ui = "0x65F1206182C77A040ED41D507B59C622FA94AB5E71CCA567202CFF3909F3D5C4DBE338E45276630FD8237149DD47EE027AF26F9C619723D0"
         self.assertEqual(
-            self.gnosis_protocol_api.get_trades(
-                "0x9c79b5883b7f2bacbedef554a835fb07c21f4b1b046edf510554a6ba0444d2665ac255889882acd3da2aa939679e3f3d4ce"
-                "a221e72eb7b80"
-            ),
+            self.mainnet_gnosis_protocol_api.get_trades(order_ui=mainnet_order_ui),
```
Comment (moisses89) [inline gnosis/protocol/tests/test_gnosis_protocol_api.py:82]: Why the change for goerli test to mainnet test?

Comment (Uxio0) [reply gnosis/protocol/tests/test_gnosis_protocol_api.py:82]: Because there's not enough liquidity on goerli and I cannot find a trade I can use
